server/testutil/testbazel/test_rules.bzl

```diff
@@ -0,0 +1,18 @@
+def _run_shell_impl(ctx):
```

Comment:
maybe we save this for a later day? I'm a little shy about customizing bazel too hard because things will break / change / require upkeep as bazel versions change.

Reply:
Sounds good - changed back to sh_binary